catroid/res/values/colors.xml

```diff
@@ -75,4 +75,10 @@
     <color name="bluetooth_device_list_divider_color" >#505050</color >
     <color name="dialog_text_line" >#696969</color >
 
+    <!-- Phiro RGBColor Chooser Colors -->
+    <color name="color_chooser_rgb_red">#FFEB4618</color>
+    <color name="color_chooser_rgb_green">#FF078707</color>
+    <color name="color_chooser_rgb_blue">#FF0284e7</color>
```

Comment:
It's a bit redundant to declare the alpha value if it is fully opaque. Also all letters should be uppercase for consistency, there's a lowercase `e`.